Btchap/target.yml

```diff
@@ -155,20 +163,33 @@ targets:
     - path: ../Riot/Modules/KeyBackup
     - path: ../Riot/Modules/KeyVerification
     - path: ../Riot/Modules/MediaPicker
+    - path: ../Riot/Modules/PublicRoomList
     - path: ../Riot/Modules/QRCode
     - path: ../Riot/Modules/Reauthentication
     - path: ../Riot/Modules/Room/BubbleReactions
     - path: ../Riot/Modules/Room/DataSources
+    - path: ../Riot/Modules/Room/NotificationSettings
+      excludes:
```

Comment:
can you remove this resources instead of exclude them?

Reply:
I removed these duplicate files in last commit.